content/docs/2.7/scalers/azure-data-explorer.md

```diff
@@ -0,0 +1,214 @@
++++
+title = "Azure Data Explorer"
+layout = "scaler"
+availability = "v2.7+"
+maintainer = "Community"
+description = "Scale applications based on Azure Data Explorer query result."
+go_file = "azure_data_explorer_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-data-explorer` trigger that scales based on an Azure Data Explorer query result.
+
+```yaml
+triggers:
+- type: azure-data-explorer
+  metadata:
+    podIdentity: "azure" # Provide either podIdentity or aad app creds. can use TriggerAuthentication as well
```

Comment:
We should leave this out as this should only be supported through TriggerAuthentication, sorry

```suggestion
```